label_studio/users/models.py

```diff
@@ -26,6 +27,10 @@
 class UserManager(BaseUserManager):
     use_in_migrations = True
 
+    def __init__(self, disable_is_deleted_filter=False, *args, **kwargs):
```

Comment:
Double negatives are a very common source of bugs, could we try to stick to positive phrasing and define this as `filter_soft_deleted: bool = True`?

Reply:
Actually with the second manager refactoring, we can get rid of this param entirely. Instead we should update the migration to use users.models.WithDeletedUserManager here https://github.com/HumanSignal/label-studio/pull/4832/files#diff-41b3a1e27082ee6c77df7e40fb9d0af80e381bf55d713ef72acd03ca6d79a4b4R71